### PR TITLE
Test primary traveler

### PR DESCRIPTION
**Pull request by simon-the-sloth**

### CODE REVIEW

File: EquiTrack/t2f/tests/test_travel_details.py [hunk 1]
```diff
@@ -510,6 +510,41 @@ def test_ta_not_required(self):
...
+                'action_points': [],
+                'ta_required': False,
+                'international_travel': False,
+                'traveler': self.traveler.id,
+                'mode_of_travel': []}
+
+        response = self.forced_auth_req('post', reverse('t2f:travels:list:index'),
+                                        data=data, user=self.unicef_staff)
+        response_json = json.loads(response.rendered_content)
+        self.assertEqual(response_json, {'activities': [{'primary_traveler': ['This field is required']}]})
```
Comment (ewheeler) [inline EquiTrack/t2f/tests/test_travel_details.py]: looks like this should be `'This field is required.'`

Missing period is the culprit for test failure: https://circleci.com/gh/unicef/etools/2953

Comment (simon-the-sloth) [inline EquiTrack/t2f/tests/test_travel_details.py]: Fixed.

**Comment by robertavram** [comment]
@simon-the-sloth which issue does this PR connect to?

**Comment by simon-the-sloth** [comment]
@robertavram Issue 191

**Comment by robertavram** [comment]
@simon-the-sloth please reference the issue as specified in etools-issues readme

**Pull request merged**